src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,107 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.lib.Dao;
+import mate.academy.lib.DataProcessingException;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String URL = "jdbc:mysql://localhost:3306/your_schema?serverTimezone=UTC";
+    private static final String USER = "your_user";
+    private static final String PASSWORD = "your_password";
+
+    @Override
+    public Book create(Book book) {
+        String insertQuery = "INSERT INTO books (title, price) VALUES (?, ?)";

```

Comment:
Remember to use uppercase for SQL keywords in your queries for better readability. For example, use `INSERT INTO` instead of `insert into`.